include/container/seadPtrArray.h

```diff
@@ -86,7 +86,12 @@ class PtrArrayImpl
 
     void pushFront(void* ptr) { insert(0, ptr); }
 
-    void* popBack() { return isEmpty() ? nullptr : mPtrs[--mPtrNum]; }
+    void* popBack()
+    {
+        if (mPtrNum >= 1)
```

Comment:
does this still match if you invert the condition?

```suggestion
        if (mPtrNum < 1)
            return nullptr;
        return mPtrs[--mPtrNum];
```

reads slightly better imo

Reply:
this causes a `sead::HeapMgr::destroy()` to no longer match SMO (seems to reorder the branches)

Reply:
welp, oh well.